build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -381,10 +417,43 @@ def run_benchmarks_for_category(
   return (results, errors)
 
 
+def get_available_drivers(tool_dir: str, verbose: bool) -> Sequence[str]:
+  config_txt_file_path = os.path.join(tool_dir, "config.txt")
+  config_txt_file = open(config_txt_file_path, "r")
+  config_txt_file_lines = config_txt_file.readlines()
+  available_drivers = []
+  for line in config_txt_file_lines:
+    cmake_var_name, cmake_var_value = line.strip().split("=")
```

Comment:
This _almost_ works with `CMakeCache.txt`. Maybe worth supporting that syntax too?

```
//Enables the 'vmvx' runtime HAL driver
IREE_HAL_DRIVER_VMVX:BOOL=ON

//Enables the 'vmvx-sync' runtime HAL driver
IREE_HAL_DRIVER_VMVX_SYNC:BOOL=ON
```

(skip comments and types)

meta point: it'd be nice if this script didn't assume _anything_ about the build environment (CMake, Bazel, installed from binary distribution, etc.)

Reply:
Renamed to drop references to `cmake`.
I locally implemented supporting `NAME:TYPE=VALUE` syntax, that was easy, but then I thought I'm adding code handling generality that isn't actually exercised at the moment. Generally I feel very reluctant to add dead code. So I undid that local change. But clearly as you say it would be easy to support directly parsing CMakeCache.txt if one wanted to.